src/buildtools/buildtools.targets

```diff
@@ -44,7 +44,7 @@
           BeforeTargets="CoreCompile">
 
     <PropertyGroup>
-        <FsYaccPath Condition="'$(FsYaccPath)' == ''">$(ArtifactsDir)\Bootstrap\fsyacc\fsyacc.dll</FsYaccPath>
+        <FsYaccPath Condition="'$(FsYaccPath)' == ''">$(ArtifactsDir)\bin\fsyacc\Release\net5.0\fsyacc.dll</FsYaccPath>
```

Comment:
@ncave Is this working for you? I had to revert changes in this file to make MSBuild work.

Reply:
@alfonsogarciacaro Yes, that works. Make sure to run the `fcs/build.sh` which builds the tooling first.

Reply:
@alfonsogarciacaro Apropos, latest FCS 2021-11-15 (which this was rebased to) has some bug fixed, so you might want to get latest/build again from this fork.